src/Extensions/Abstractions/Build/Microsoft.Omex.Extensions.Abstractions.props

```diff
@@ -0,0 +1,22 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Default properties for omex projects -->
+  <PropertyGroup>
+    <!-- Enforce nulability check and language version -->
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <!-- Warnings as errors -->
+    <ErrorReport>prompt</ErrorReport>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+    <!-- Enforce checks -->
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <!-- Define common properties -->
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <CurrentYear>$([System.DateTime]::Now.ToString(yyyy))</CurrentYear>
```

Comment:
>Now [](start = 38, length = 3)

Should this be `UtcNow`, rather than build timezone dependent?

Reply:
removed this logic